mockwebserver/src/main/kotlin/mockwebserver3/MockWebServer.kt

```diff
@@ -939,10 +939,10 @@ class MockWebServer : Closeable {
       }
 
       when (socketPolicy) {
-        DISCONNECT_AT_END -> {
+        DisconnectAtEnd -> {
           stream.connection.shutdown(ErrorCode.NO_ERROR)
         }
-        DO_NOT_READ_REQUEST_BODY -> {
+        DoNotReadRequestBody -> {
```

Comment:
Seems a shame to not put things like http2ErrorCode into the DoNotReadRequestBody type?